packages/example/src/modules/program/view/ProgramList.tsx

```diff
@@ -45,46 +50,67 @@ export const ProgramList = ({
 
   const programInfos = useMemo(() => getPrograms(1000), []);
 
+  useEffect(() => {
+    const remoteControlListener = (pressedKey: SupportedKeys) => {
+      if (
+        pressedKey !== SupportedKeys.Back ||
+        !isActive ||
+        listRef.current.currentlyFocusedItemIndex === 0
+      ) {
+        return false;
+      }
+      listRef.current.focus(0);
+      return true;
+    };
+    RemoteControlManager.addKeydownListener(remoteControlListener);
+
+    return () => RemoteControlManager.removeKeydownListener(remoteControlListener);
+  }, [isActive, listRef, navigation]);
```

Comment:
Maybe the time has come to create a hook `useKey(SupportedKeys.Back, () => { yourWholeLogic() })` 😁

Reply:
Can I use a custom hook like this in a useEffect? :o Interested to discuss this

Reply:
Done :-)